libmscore/textbase.cpp

```diff
@@ -192,6 +192,22 @@ void TextCursor::setFormat(FormatId id, QVariant val)
       text()->setTextInvalid();
       }
 
+//---------------------------------------------------------
+//   makeCharReadable
+//---------------------------------------------------------
+
+QString makeCharReadable(QString chr)
+      {
+      if (chr.isEmpty() || chr == "\0")
+            return "blank";
+      Q_ASSERT(chr.length() == 1);
+      if (chr == " ")  return "space";
+      if (chr == "\r") return "carriage return";
+      if (chr == "\n") return "line feed";
+      if (chr == ";")  return "semicolon";
+      return chr;
+      }
```

Comment:
@MarcSabatella, unfortunately NVDA doesn't read the special characters when they are on their own, hence this function to handle them manually. I don't suppose you can think of a better way to do this?

Reply:
In know NVDA (like most other screenreaders) has options to control verbosity of punctuation, but I don't know how it works in this context.  I'm away from my Windows computer until Monday but will play with it more then.  As it is, I can't think of a better way to handle this offhand.